### ISSUE TITLE
Crashes every time

### ISSUE THREAD

**Issue by jab416171** [KoalaBear84/OpenDirectoryDownloader#19]
**Describe the bug**
Crashing with stack trace

**To Reproduce**
1. Execute ./OpenDirectoryDownloader --url https://download.tuxfamily.org/

**Expected behavior**
No failure

```Fatal error. Internal CLR error. (0x80131506)
Fatal error. Internal CLR error. (0x80131506)
   at System.Diagnostics.StackTrace.GetStackFramesInternal(System.Diagnostics.StackFrameHelper, Int32, Boolean, System.Exception)
   at System.Diagnostics.StackFrameHelper.InitializeSourceInfo(Int32, Boolean, System.Exception)
   at System.Diagnostics.StackTrace.CaptureStackTrace(Int32, Boolean, System.Exception)
   at NLog.LoggerImpl.Write(System.Type, NLog.Internal.TargetWithFilterChain, NLog.LogEventInfo, NLog.LogFactory)
   at OpenDirectoryDownloader.OpenDirectoryIndexer+<WebDirectoryProcessor>d__53.MoveNext()
   at System.Threading.ExecutionContext.RunInternal(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object)
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1+AsyncStateMachineBox`1[[System.Threading.Tasks.VoidTaskResult, System.Private.CoreLib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e],[System.__Canon, System.Private.CoreLib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]].MoveNext(System.Threading.Thread)
   at System.Threading.Tasks.AwaitTaskContinuation.RunOrScheduleAction(System.Runtime.CompilerServices.IAsyncStateMachineBox, Boolean)
   at System.Threading.Tasks.Task.RunContinuations(System.Object)
   at System.Threading.Tasks.Task.TrySetResult()
   at System.Threading.Tasks.Task+DelayPromise.CompleteTimedOut()
   at System.Threading.TimerQueueTimer.CallCallback(Boolean)
   at System.Threading.TimerQueueTimer.Fire(Boolean)
   at System.Threading.TimerQueue.FireNextTimers()
Aborted (core dumped)
```

**Desktop (please complete the following information):**
 - OS: Archlinux
 - Version OpenDirectoryDownloader 1.0.0

**Comment by KoalaBear84** [comment]
Hi Joseph,

Thanks for your report. I have never seen this error. Do you get this immediately? (Press S a couple of times to see if it does scan something).

Can you try another URL?

Do you use the version of the releases page? If you are able to, you could clone the current source, install .NET Core 3.1 SDK and run it like `dotnet run -- --url https://download.tuxfamily.org/`, in the OpenDirectoryDownloader directory (if I'm correct, or run without parameters and fill in the URL.

**Comment by KoalaBear84** [comment]
I've tried to scan it locally. So the problem is not in the Open Directory. Hopefully you can supply more info about it.

|**Url:** https://download.tuxfamily.org/||[Urls file](https://gofile.io/?c=H6xyuH)|
|:-|-:|-:|
|**Extension (Top 5)**|**Files**|**Size**|
|.ogg|2,619|93.88 GiB|
|.iso|211|70.58 GiB|
|.zip|4,530|57.66 GiB|
|.avi|441|44.9 GiB|
|.tazpkg|39,756|35.43 GiB|
|**Dirs:** 164,283 **Ext:** 1,121|**Total:** 807,797|**Total:** 636.25 GiB|
|**Date (UTC):** 2020-02-17 05:39:31|**Time:** 01:44:57|**Speed:** 30.6+ MB/s (245+ mbit)|

**Comment by KoalaBear84** [comment]
Thanks @snakefoot

Didn't saw this, thanks. So, with the newest release it should be automatically fixed as this already haves NLog 4.7.

Let me know if it is fixed. If you still have an issue, reopen it.
